### PR TITLE
operator: add new command operator

### PR DESCRIPTION
**Pull request by subhamkrai**
adding new command operator which calls sub-command rook which calls sub-command set.

```
 tree
.
├── operator.go
└── rook
    ├── rook.go
    └── set.go
```

### CODE REVIEW

File: cmd/odf/operator/rook/set.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import (
+	"github.com/red-hat-storage/odf-cli/cmd/odf/root"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/spf13/cobra"
+)
+
+var setCmd = &cobra.Command{
+	Use:     "set",
+	Short:   "Set the property in the rook-ceph-operator-config configmap.",
+	Example: "odf operator rook set <KEY> <VALUE>",
```
Comment (subhamkrai) [inline cmd/odf/operator/rook/set.go:12]:
```suggestion
	Example: "odf operator rook set <KEY> <VALUE>",
```

@travisn does this command structure sounds good?

Comment (travisn) [reply cmd/odf/operator/rook/set.go:12]: Yes, lgtm

File: cmd/odf/operator/operator.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import (
+	"github.com/red-hat-storage/odf-cli/cmd/odf/operator/rook"
+	"github.com/spf13/cobra"
+)
+
+// OperatorCmd represents the rook commands
+var OperatorCmd = &cobra.Command{
+	Use:   "operator",
+	Short: "Calls subcommands for 'operator'",
```
Comment (travisn) [inline cmd/odf/operator/operator.go:11]:
```suggestion
	Short: "Calls subcommands specific to various ODF operators",
```

File: cmd/odf/operator/rook/rook.go [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package rook
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// RookCmd represents the rook commands
+var RookCmd = &cobra.Command{
+	Use:   "rook",
+	Short: "Calls subcommands for 'rook'",
```
Comment (travisn) [inline cmd/odf/operator/rook/rook.go:10]:
```suggestion
	Short: "Calls subcommands for the rook operator",
```

**Review by travisn — CHANGES_REQUESTED** [review]
Just a couple small text suggestions

File: cmd/odf/operator/rook/rook.go (hunk 3 above)
Comment (subhamkrai) [reply cmd/odf/operator/rook/rook.go]: done

File: cmd/odf/operator/operator.go (hunk 2 above)
Comment (subhamkrai) [reply cmd/odf/operator/operator.go]: done

File: cmd/odf/operator/rook/set.go (hunk 1 above)
Comment (subhamkrai) [reply cmd/odf/operator/rook/set.go:12]: great

File: README.md [hunk 4]
```diff
@@ -16,6 +16,9 @@ The ODF CLI tool provides configuration and troubleshooting commands for OpenShi
 - `odf subvolume delete <subvolume> <filesystem> <subvolumegroup>`: Deletes the stale subvolumes
 - `odf mons`
   - `restore-quorum <monID>`: Restore the mon quorum based on a single healthy mon since quorum was lost with the other mons
+- `odf operator`:
+  - `rook`:
```
Comment (subhamkrai) [inline README.md:20]:
```suggestion
  - `rook`:
```

@travisn should I add restart sub-command to restart rook operator

Comment (travisn) [reply README.md:20]: Yes, makes sense to me

**Review by travisn — APPROVED** [review]

**Pull request merged**